docs/cis_self_assessment.md

```diff
@@ -355,225 +351,106 @@ root:root
 
 #### 1.1.18
-Ensure that the `controller-manager.conf` file ownership is set to `root:root` (Scored)
+Ensure that the `controller.kubeconfig` file ownership is set to `root:root` (Scored)
 <details>
 <summary>Rationale</summary>
-The controller-manager.conf file is the kubeconfig file for the Controller Manager. You should set its file ownership to maintain the integrity of the file. The file should be owned by root:root.
+The controller.kubeconfig file is the kubeconfig file for the Scheduler. You should set its file ownership to maintain the integrity of the file. The file should be owned by root:root.
+
+In RKE2, this file is located at `/var/lib/rancher/rke2/server/cred/controller.kubeconfig`.
 </details>
 
-**Result:** Not Applicable
+**Result:** Pass
 
-**Remediation:**
-RKE doesn’t require or maintain a configuration file for the controller manager. All configuration is passed in as arguments at container run time.
+**Audit:**
+```bash
+stat -c %U:%G /var/lib/rancher/rke2/server/cred/controller.kubeconfig
+root:root
+```
 
+**Remediation:**
+RKE2 creates this file at `/var/lib/rancher/rke2/server/cred/controller.kubeconfig` and automatically sets its ownership to `root:root`.
 
 #### 1.1.19
 Ensure that the Kubernetes PKI directory and file ownership is set to `root:root` (Scored)
 <details>
 <summary>Rationale</summary>
-Kubernetes makes use of a number of certificates as part of its operation. You should set the ownership of the directory containing the PKI information and all files in that directory to maintain their integrity. The directory and files should be owned by root:root.
+RKE2 makes use of a number of certificates as part of its operation. You should set the ownership of the directory containing the PKI information and all files in that directory to maintain their integrity. The directory and files should be owned by root:root.
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chown -R root:root /etc/kubernetes/ssl
-```
-
 **Audit:**
-
-```
-stat -c %U:%G /etc/kubernetes/ssl
+```bash
+stat -c %U:%G /var/lib/rancher/rke2/server/tls
+root:root
 ```
 
-**Expected result**:
-
-```
-'root:root' is present
+**Remediation:**
+```bash
+chown -R root:root /var/lib/rancher/rke2/server/tls
 ```
 
 
 #### 1.1.20
 Ensure that the Kubernetes PKI certificate file permissions are set to `644` or more restrictive (Scored)
 <details>
 <summary>Rationale</summary>
-Kubernetes makes use of a number of certificate files as part of the operation of its components. The permissions on these files should be set to 644 or more restrictive to protect their integrity.
+RKE2 makes use of a number of certificate files as part of the operation of its components. The permissions on these files should be set to 644 or more restrictive to protect their integrity. This is a manual check and each file in the output will need to be verified.
```

Comment:
> This is a manual check and each file in the output will need to be verified.

I don't think you need to add this here. This guide is basically all manual checks. My hope is that the rationale section can be mostly copy/pasted from the CIS guide, with just minimal tweaks to make specific to rke2.

I get what you're trying to do though. Maybe my audits in the first 10 entries were too terse as compared to the cis guide...we could add what someone should be looking for to the audit section, like the guide does.

Like for this control, the audit section could be:
**Audit:**

Run the below command on your server nodes:
```bash
stat -c %a\ %n /var/lib/rancher/rke2/server/tls/*.crt
```
Verify that the permissions are 644 or more restrictive.